Services/LDAP/classes/class.ilLDAPRoleAssignmentTableGUI.php

```diff
@@ -19,7 +19,7 @@
  */
 class ilLDAPRoleAssignmentTableGUI extends ilTable2GUI
 {
-    public function __construct($a_parent_obj, $a_parent_cmd = '')// TODO PHP8-Review Type hints are missing here
+    public function __construct(?object $a_parent_obj, string $a_parent_cmd = '')
```

Comment:
IMO you can assume that the first paremeter MUST not be `null`. You can remove the `?`.